userspace/libsinsp/sinsp.h

```diff
@@ -71,6 +71,7 @@ limitations under the License.
 #include <libsinsp/sinsp_suppress.h>
 #include <libsinsp/state/table_registry.h>
 #include <libsinsp/stats.h>
+#include <libsinsp/stats_v2_resource_utilization.h>
```

Comment:
@federico-sysdig suggestion instead of touching the stats and adding a new header file, would it be ok to move those changes to the refactor PR we have here: https://github.com/falcosecurity/libs/pull/1652 Open to however you would like to go about it.

Reply:
Absolutely. I am open to several ways to proceed. If you want to integrate these changes into the other PR, I can close this as superseded. I have no interest on the "paternity" of this change.

Reply:
Great let's do this then. If you leave this commit and add another one on top we can come back to it later in the other PR. Ty!

Reply:
I'm lost. What should I do? I can let this PR lag behind the other one and merge later on. Or you can integrate some of these changes there. Or what?

Reply:
If ok, revert anything that touched the stats, but ok to leave it in the history. I can port them over to the other PR later. That way we do not block all other changes here and keep the larger metrics refactor in one place.

Reply:
There is an array initializer in `stats.cpp` that uses c99 array designators, if I revert that I also need to revert the warning suppression and the PR would lose its purpose and a second PR would be needed.
This PR isn't critical. It can wait as much time as #1652 needs to be merged. I'll then patch this up for the new situation and ask you to review. Take your time, I've put this as draft for the moment.

Reply:
ok let's do this then, ty!